examples/kubernetes/1.10/cilium-ds.yaml

```diff
@@ -45,6 +45,11 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
+        - name: CILIUM_IDENTITY_ALLOCATION_MODE
```

Comment:
This should not be needed, cilium will read the option from the `config-dir=/tmp/cilium/config-map`

Reply:
oh, hmm. I'll try it out but I'm sure you are correct.